Assets/Plugins/Source/Editor/EpicOnlineServicesConfigEditor.cs

```diff
@@ -225,269 +212,6 @@ private void SaveToJSONConfig(bool prettyPrint)
             AssetDatabase.Refresh();
         }
 
-        public static string CollectFlags(List<string> flags)
-        {
-            return String.Join("|", EmptyPredicates.NewIfNull(flags));
-        }
-
-        public static List<string> SplitFlags(string collectedFlags)
-        {
-            return new List<string>(collectedFlags.Split('|'));
-        }
```

Comment:
These one-line functions were only ever referenced in one place apiece, so I removed them - replacing them with the function body.